.github/workflows/go.yml

```
@@ -60,13 +66,70 @@ jobs:
-      #   uses: codecov/codecov-action@v4
-      #   with:
-      #     token: ${{ secrets.CODECOV_TOKEN }}
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+  benchmark:
+    name: Benchmark
+    runs-on: ubuntu-latest
+    if: false
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)